src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.html

```diff
@@ -2,7 +2,13 @@ <h1 mat-dialog-title>GitHub App Logs</h1>
 <div mat-dialog-content>
   <app-loading [loading]="loading">
     <mat-card *ngIf="showContent === 'error'" class="alert alert-warning mat-elevation-z" role="alert">
-      <mat-icon class="alert-warning-icon">warning</mat-icon> There were problems retrieving GitHub App logs for this organization.
+      <mat-icon class="alert-warning-icon">warning</mat-icon> There were problems retrieving the GitHub App logs for this organization.<br />
+      Typically, if you cannot view the logs for your GitHub user account, you must unlink and relink it in<br />
+      your Dockstore profile.<br />
+      Otherwise, your GitHub user or the Dockstore app may not have access to the GitHub organization.<br />
+      <a href="https://github.com/settings/connections/applications/7ad54aa857c6503013ea" target="_blank" rel="noopener noreferrer"
```

Comment:
1. Interesting, I hadn't seen that page before, and I don't know how you found it :). I can't find a link to it in the GitHub UI; I can only navigate directly there via your URL.
2. It would be nice if the app id weren't hard-coded, but I guess it only affects our staging and qa environments -- I don't know how much effort it's worth.

Reply:
1. If you asked my mom, she would say its because I'm a genius, but actually, it appears here in the current UI: https://github.com/dockstore/dockstore-ui2/blob/162afe44fdbb783b92ceb384610f71a408da5fd7/src/app/myworkflows/my-workflow/my-workflow.component.html#L104

2. Yeah, I'm inclined to leave as-is unless someone feels strongly that we should make it work for staging and qa.